src/main.ts

```diff
@@ -161,6 +164,9 @@ const readPlexPrefsIfNeeded = async () => {
 			LetterboxdPlugin,
 			RequestsPlugin,
 			DashboardPlugin,
+			SectionHubsPlugin,
+			JustWatchPlugin,
+			PlayRedirectPlugin,
```

Comment:
please make separate PRs for this when you get the chance. There's too much functionality in this one PR and I'd like to review this stuff separately